### PR TITLE
Package promotion related changes

### PR DESCRIPTION
**Pull request by sir-sigurd**
# Description

# TODO

- [x] Unit tests
- [x] <del>Documentation</del>
    - [x] <del>[Run `build.py`](../gendocs/build.py) for new docstrings</del>
- [x] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

**Comment by sir-sigurd** [comment]
@fiskus do [schema changes](api/python/quilt3/workflows/config-1.schema.json) look fine for you?

**Comment by fiskus** [comment]
Yeah, I use this format on frontend and think it's fine

File: api/python/quilt3/workflows/config-1.schema.json [hunk 1]
```diff
@@ -72,6 +84,25 @@
           }
         }
       }
+    },
+    "successors": {
+      "type": "object",
```
Comment (akarve) [inline api/python/quilt3/workflows/config-1.schema.json:89]: TODO: add description that these are buckets, etc.

**Comment by sir-sigurd** [comment]
@nl0  @fiskus 
I asked for review mostly for language in changelog and schema.

**Review by nl0 — APPROVED** [review]
lgtm

File: api/python/quilt3/workflows/config-1.schema.json [hunk 2]
```diff
@@ -72,6 +72,36 @@
...
+      "description": "Buckets usable as destination with \"Push to Bucket\" in web UI.",
```
Comment (fiskus) [inline api/python/quilt3/workflows/config-1.schema.json:78]: as **a** destination

---

Just noting, I use free Grammarly for this: 
Screenshot from 2020-12-01 16-48-11

**Review by fiskus — APPROVED** [review]

**Pull request merged**